### ISSUE TITLE
Add form example

### ISSUE THREAD

**Issue by benrudolphWSP** [WSP-Denver/project-skeleton-sass#6]
Add a simple form to show what a form would look like and how it should be structured semantically and for accessibility.

Should this be built as a component? Kind of like I did for the [simple modal](https://github.com/WSP-Denver/simple-modal)?
